tests/unit/package_managers/bundler/test_parser.py

```diff
@@ -203,3 +203,54 @@ def test_dependencies_could_be_downloaded(
     dependency.download_to(base_destination)
 
     mock_downloader.assert_called_once_with(expected_source_url, expected_destination)
+
+
+@mock.patch("cachi2.core.package_managers.bundler.parser.Repo.clone_from")
+def test_download_git_dependency_works(
+    mock_git_clone: mock.Mock,
+    rooted_tmp_path: RootedPath,
+    caplog: pytest.LogCaptureFixture,
+) -> None:
+    dep = GitDependency(
+        name="example",
+        version="0.1.0",
+        url="https://github.com/user/repo.git",
+        ref="f" * 40,
```

Comment:
I've seen this elsewhere already, don't we want a constant for this? e.g. `TEST_REF="f"*40`